tools/sil-opt/SILOpt.cpp

```diff
@@ -519,13 +525,23 @@ int main(int argc, char **argv) {
   SILOpts.EnableOSSAModules = EnableOSSAModules;
   SILOpts.EnableCopyPropagation = EnableCopyPropagation;
   SILOpts.DisableCopyPropagation = DisableCopyPropagation;
-  SILOpts.EnableExperimentalLexicalLifetimes =
-      EnableExperimentalLexicalLifetimes;
-  // Also enable lexical lifetimes if experimental move only is enabled. This is
-  // because move only depends on lexical lifetimes being enabled and it saved
-  // some typing ; ).
-  SILOpts.EnableExperimentalLexicalLifetimes |=
-    EnableExperimentalMoveOnly;
+
+  // Enable lexical lifetimes if it is set or if experimental move only is
+  // enabled. This is because move only depends on lexical lifetimes being
+  // enabled and it saved some typing ; ).
+  bool enableExperimentalLexicalLifetimes =
+      EnableExperimentalLexicalLifetimes | EnableExperimentalMoveOnly;
+  if (enableExperimentalLexicalLifetimes && DisableLexicalLifetimes) {
+    fprintf(
```

Comment:
nit: `llvm::errs() << ...`

Reply:
I was following what was happening in the rest of the file.

Reply:
OK, as I said, neither comment is really too important.